### PR TITLE
Quality of life improvements

### PR DESCRIPTION
**Pull request by Lucki**
This adds some quality of life improvements to the daily quests.

Some can be done automatically:
* Swap badge
* Swap border
* Share content
* Visit news

Some can be improved to solve manually:
* Swap avatar -> quick link to personalization site (currently untested/placeholder since the quest is quite rare)
* Post replies -> quick link to the AWA On-Topic forum section
* Visit page -> quick link to the AWA On-Topic forum section

___

I haven't done something with JavaScript before so let me know when there's something to improve.

P.S.: I have no idea what changed in the first line, I copied the old line in this and git still tells me there has something changed.

### CODE REVIEW

**Comment by thomas-ashcraft** [comment]
> P.S.: I have no idea what changed in the first line, I copied the old line in this and git still tells me there has something changed.

Most likely this is UTF BOM. Shouldn't really affect functionality in our case. Can be a problem only when you creating something that working with files in straight ways.

**Comment by thomas-ashcraft** [comment]
This is awesome! Automatic daily quests resolving - this is what I dreamt to make. You made that dream come true. Thank you!

Not immediately able to find a button to solve the quest. I think we need to make it a little more noticeable when the quest is not yet complete.

Script messages aren't visible with opened sidebar because script UI is lies under it.

But those notes relates to script UI design for the new site interface, which is whole dedicated task that I can't find a time for yet.

**Review by thomas-ashcraft — APPROVED** [review]

File: alienwarearena_helper.user.js [hunk 1]
```diff
@@ -595,6 +595,183 @@ Sorting from fresh ones to old ones.">Vote for newly uploaded ${sectionType}${(s
...
+			await new Promise((r) => setTimeout(r, 500));
+		}
+
+		try {
+			let response = await getURL("/api/v1/users/arp/status");
+			console.log("👽 QUEST: " + response.quests[0].title);
+			switch (response.quests[0].type) {
+				case "change_border":
+					$(`<a class="btn btn-default awah-btn-quest" href="javascript:void(0);" data-awah-tooltip="Automatic border swap" data-awah-quest="border">
+						<span class="more-link right"></span></a>`).appendTo(".col-2");
```
Comment (thomas-ashcraft) [inline alienwarearena_helper.user.js]: ```.col-2``` can be a common selector. There is better to make something like ```.quest-item > .col-2```

**Review by thomas-ashcraft — CHANGES_REQUESTED** [review]

**Comment by Lucki** [comment]
Thanks!

**Comment by thomas-ashcraft** [comment]
Have made a little improvement to set previous avatar border back in automatic way. ( https://github.com/thomas-ashcraft/alienware_arena_helper/commit/74b380f69629683f8064f0d486fb75d200afaf7e )

Looking forward to make the same for badges.

**Pull request merged**